lang/en.yml

```diff
@@ -33,6 +33,12 @@ en:
     CollapsePanel: 'Collapse panel'
     DELETED: Deleted.
     DropdownBatchActionsDefault: 'Choose an action...'
+    ErrorDetail: 'Error detail: {detail}'
+    ErrorMessage: '<p>Sorry, it seems there was a {errorcode} error.</p>'
+    ErrorMessage403: '<p>Sorry, it seems the action you were trying to perform is forbidden.</p>'
+    ErrorMessage404: '<p>Sorry, it seems you were trying to access a page that doesn''t exist.</p>'
```

Comment:
It's not really a page that does not exist - it's a route that does not exist.  Maybe a "section" is better language?

e.g. `/admin/security/asdfasdfsadf` is will say this page does not exist, though you wouldn't call `/admin/security` a page

Reply:
True. I think "section" has the same problem as it may be for example a specific ID in an edit form that has no associated data object.
How about something like "You were trying to access a section or object that doesn't exist"? Or simply "The URL you were trying to access resulted in a 'Not Found' error"?